client/my-sites/media-library/list.jsx

```diff
@@ -1,14 +1,15 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
+var React = require( 'react' ),
 	clone = require( 'lodash/clone' ),
 	noop = require( 'lodash/noop' ),
 	filter = require( 'lodash/filter' ),
 	findIndex = require( 'lodash/findIndex' );
 
+import { AutoSizer, InfiniteLoader, Grid, WindowScroller } from 'react-virtualized';
 import { connect } from 'react-redux';
+import fill from 'lodash/fill';
```

Comment:
With regards to my other comment, we could enable root imports by adding to our [`babel-transform` configuration](https://github.com/Automattic/wp-calypso/blob/master/.babelrc):

```
"react-virtualized": {
	"transform": "react-virtualized/${member}"
}
```

See: https://www.npmjs.com/package/babel-plugin-transform-imports